src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -52,7 +53,14 @@ internal abstract partial class PatternMatcher : IDisposable
             bool allowFuzzyMatching = false)
         {
             culture = culture ?? CultureInfo.CurrentCulture;
+            if (culture.Name == "en-US")
+            {
+                culture = CultureInfo.InvariantCulture;
```

Comment:
Um, InvariantCulture is the same as en-US but with outdated rules. Can you clarify the fast paths here?